lib/unleash/strategy/flexible_rollout.rb

```diff
@@ -10,9 +10,10 @@ def name
       # need: params['percentage']
       def is_enabled?(params = {}, context = nil)
         return false unless params.is_a?(Hash)
-        return false unless context.instance_of?(Unleash::Context)
 
         stickiness = params.fetch('stickiness', 'default')
+        return false unless context_is_sufficient?(stickiness, context)
```

Comment:
Any way we can simplify further by turning the `unless` to a `if` ?

Would a name like `context_required?`/`context_needed?`/`context_valid?`/`context_invalid?` be more descriptive?

Reply:
ok i can change it